aries_cloudagent/config/argparse.py

```diff
@@ -545,6 +556,27 @@ def get_settings(self, args: Namespace) -> dict:
         settings = {}
         if args.external_plugins:
             settings["external_plugins"] = args.external_plugins
+
+        if args.plugin_config:
+            with open(args.plugin_config, "r") as stream:
+                plugins_conf = yaml.safe_load(stream)
+
+            if not settings.get("external_plugins"):
+                settings["external_plugins"] = []
+
+            settings["plugins_config"] = {}
+            for plugin in plugins_conf.get("plugins", []):
+                plug_dir = plugin.get("local_directory")
```

Comment:
Why is `local_directory` required?

Reply:
In the document, I considered that there could be many entry points to load the plugin https://hackmd.io/ROUzENdpQ12cz3UB9qk1nA#Possible-improvements

However, I currently think that the solution should be the more simple as possible, and later on, if we need more inputs, we can redefine it.